sklearn/tests/test_metaestimators.py

```diff
@@ -54,10 +58,31 @@ def __init__(self, name, construct, skip_methods=(),
                   lambda est: SelfTrainingClassifier(est),
                   skip_methods=['transform', 'inverse_transform',
                                 'predict_proba']),
+    DelegatorData(
+        'Pipeline_with_NestedMetaEstimator',
+        lambda est: Pipeline([
+            ('stacking',
+             StackingClassifier([('clf', LogisticRegression())],
+                                final_estimator=est))
+        ]),
+        skip_methods=["transform", "inverse_transform", "predict_log_proba",
+                      "score"],
+    ),
+    DelegatorData(
+        'ClassifierChain_with_NestedMetaEstimator',
```

Comment:
For reviewer: a case where the outer meta-estimator will create a list of fitted inner meta-estimator.

Reply:
We rely on delegating to `estimators_`